Makefile

```diff
@@ -26,3 +26,9 @@ watch: bundle
 
 start: bundle
 	npm run start
+
+redis-cli:
+	docker-compose run redis redis-cli -h redis
+
+bash:
+	docker-compose run server bash
```

Comment:
Worth adding `--rm` to all the ephemeral container we start.

Reply:
Good idea.